dev/js/wow_wizard.js

```diff
@@ -378,6 +378,30 @@
 			}
 		}
 
+		// If the step has steps depending on it, shows alert and clear the future steps.
+		function _clearFutureSteps($step) {
+			var depending_steps = [];
+			
+			for(var i = currentStep+1; i <= passedStepTracker; i++) {
+				var step_filled = wizard.settings.steps[i];
+				if(step_filled.isDependent) {
+					var found = false;
+					for(var j = 0; j < step_filled.steps.length && !found; j++) {
+						if(step_filled.steps[j].triggerStep == currentStep) {
+							depending_steps.push(i);
+							found = true;
+						}
+					}
+				}
+			}
+			if(depending_steps.length > 0) {
+				var message = (depending_steps.length == 1)?'Step ':'Steps ';
+				message += depending_steps + ' will change.'; 
+				alert(message);
+				passedStepTracker = depending_steps[0];
```

Comment:
@yigitozkavci I think that is better to set ``` depending_steps[0]; ``` because in the example1, when you fill step 5 and later change step 3, step 4 has no dependencies and it is answered and checked, you should be able to go directly to step 5 again.
On the other hand I realised that alert shows the steps that will change but now is showing their id's (step n - 1), not the user numeration. For this reason I will change alert message that prints array of depending_steps to ``` depending_steps[i] + 1 ```.